src/features/page_view_event/aggregate/index.js

```diff
@@ -104,20 +103,18 @@ export class Aggregate extends AggregateBase {
       }
     }
 
-    try { // PVTiming sends these too, albeit using web-vitals and slightly different; it's unknown why they're duplicated, but PVT should be the truth
-      var entries = globalScope.performance.getEntriesByType('paint')
-      entries.forEach(function (entry) {
-        if (!entry.startTime || entry.startTime <= 0) return
+    try {
+      queryParameters.fp = String(firstPaint.value.current)
+      queryParameters.fcp = String(firstContentfulPaint.value.current)
```

Comment:
I don't think the wrapping in `String()` is necessary. If these are query parameters, they will always be sent as a string.